actpool/actpool.go

```diff
@@ -283,12 +283,12 @@ func (ap *actPool) checkSelpWithoutState(ctx context.Context, selp *action.Seale
 	}
 
 	// Reject action if the gas price is lower than the threshold
-	if selp.Encoding() != uint32(iotextypes.Encoding_ETHEREUM_UNPROTECTED) && selp.GasPrice().Cmp(ap.cfg.MinGasPrice()) < 0 {
```

Comment:
how about backword(non-eip1559 tx) compatibility

Reply:
for non-eip1559 tx, it has `gasTipCap == nil` and will return `GasPrice()`, it is backward-compatible